tools/circle_plus_gen/lib/circle_plus.py

```diff
@@ -37,7 +37,7 @@ def get_train_param(self) -> typing.Union[TrainParam, None]:
             return None
 
         for m in metadata:
-            if m.name.decode("utf-8") == self.TINFO_META_TAG:
+            if m.name == self.TINFO_META_TAG:
```

Comment:
Before 
  *  `m.name` sometimes bytes(when reading it from file) and sometimes string(set it from json file). 

After
  * `m.name` always bytes type

Reply:
I see that `self.TINFO_META_TAG` is `bytes`. Then which diff in this PR makes `m.name` to `bytes` always?